aws/rds/test_rds_db_instance_storage_type_not_io1.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from aws.rds.helpers import get_db_instance_id
+from aws.rds.resources import rds_db_instances_with_tags
+
+
+@pytest.mark.rds
+@pytest.mark.parametrize(
+    "rds_db_instance", rds_db_instances_with_tags(), ids=get_db_instance_id,
+)
+def test_rds_db_instance_storage_type_not_io1(rds_db_instance):
```

Comment:
@bqbn Could we add a docstring here with a short description as to why we would want to check for this?

Reply:
Done.